community/kernel/src/test/java/org/neo4j/kernel/builtinprocs/BuiltInProceduresTest.java

```diff
@@ -165,6 +165,8 @@ public void shouldListSystemComponents() throws Throwable
         ) );
     }
 
+    // TODO: add unit test for dbms.changePassword procedure
```

Comment:
There are already unit test-like mocked checks in BuiltInProceduresIT for dbms.changePassword that tests at a lower level than AuthProceduresTest